extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/conf/RadiusAuthenticationProtocol.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Oops - Netbeans default template snuck in. I think this may also fail the build when the RAT plugin kicks in.

Reply:
I think this is good in most recent code.